### PR TITLE
Feature/complayout

### LINKED ISSUE #7: Build out Comp
When the page loads, we should see:
- a title, two inputs, and a button
- a page layout and colors that match the comp above

Image

### PR DESCRIPTION
**Pull request by butlertree**
# Description
Includes:
- The initial set up of the HTML head and body to closely resemble comp.   
- The initial setup in CSS of the spacing, font and color scheme to closely math comp.

# Contributors
@butlertree 

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My PR has a clear description of all proposed changes.
- [x] My PR denotes any/all team members who contributed to it.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] feat: My code is clean and readable, so my teammates can easily review it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (ericahagle) [inline index.html]: @butlertree This blank line should be removed. Per the HTML Style Guide, we shouldn't have any whitespace between lines in HTML.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>IdeaBox</title>
+</head>
+
```
Comment (ericahagle) [inline index.html]: @butlertree This blank line should also be removed.

**Comment by ericahagle** [comment]
@butlertree I haven't had a chance to pull this down and do a thorough review (@ben-rosner-williamsburg will probably get to that before me in the morning) but I mentioned a couple of formatting boo-boos that should be addressed. I also made a couple of little edits to your PR description, just to make it a little neater. I hope that's ok!

If you don't have any notes to add in the description part of your PR, feel free to just delete that section. Same goes for the checkboxes for `fix` and `refactor`—I took those out for you already, but yeah, no need for them in this PR since it's a `feat`.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 10px;
+}
+
+label {
+    flex-basis: 20%;
+    text-align: right;
+    margin-right: 10px;
+    font-size: x-large;
```
Comment (ben-rosner-williamsburg) [inline styles.css:74]: Feel as if we should probably stick to px for font size.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+button {
+    background-color: #1F1F3C;
+    color: #FFFFFF;
+    padding: 10px 20px;
+    border: none;
+    cursor: pointer;
+    text-align: center;
+    display: block;
+    height: 60px;
+    font-size: large;
```
Comment (ben-rosner-williamsburg) [inline styles.css:110]: Feel as if we should probably stick to px for font size.

**Review by ben-rosner-williamsburg — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ericahagle) [reply index.html]: Great job, @butlertree! ✅

File: index.html (hunk 2 above)
Comment (ericahagle) [reply index.html]: Great job again, @butlertree ! ✅

File: styles.css (hunk 3 above)
Comment (butlertree) [reply styles.css:74]: If I leave large, x large these font sizes are able to change relative to the size of the container they are in including changing screen sizes.  I could also gain that functionality another way but I am afraid I might start reworking all of the CSS in doing so.

File: styles.css (hunk 4 above)
Comment (butlertree) [reply styles.css:110]: If I leave large, x large these font sizes are able to change relative to the size of the container they are in including changing screen sizes.  I could also gain that functionality another way but I am afraid I might start reworking all of the CSS in doing so.

Comment (ben-rosner-williamsburg) [reply styles.css:110]: @butlertree Ah got it. Let's keep it then.

**Review by ben-rosner-williamsburg — APPROVED** [review]
As per, @butlertree 's comment, we should be fine with the CSS. Approved.

**Pull request merged**